content/en/getting_started/synthetics/api_test.md

```diff
@@ -0,0 +1,62 @@
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test:
+    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, which is a test web application.
+    - Select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
+    - Add some tags to help organize and filter tests.
+    - Select locations for testing.
+    - Click the **Test URL** button.
+
+#### Define your alert conditions
+
+Now that the test is completed, if you did not create any [assertions][4], they are automatically populated. API tests require at least one assertion to be monitored by Datadog. An assertion is defined by a parameter, an optional property, a comparator, and a target value.
+
+In this example, three default assertions are populated when testing the URL:
+
+{{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure" responsive="true" style="width:90%;">}}
+
+These assertions define the alert condition and can be customized. To add a custom assertion, click on any response header in the response preview. You can also click the **New Assertion** button to add an assertion manually (e.g., `body` contains `Shop.ist`.)
+
+{{< img src="getting_started/synthetics/api-test-configuration.gif" alt="Browser test failure" responsive="true" style="width:90%;">}}
+
+To configure the manual retry of a test when it fails, set:
+```
+An alert is triggered if your test fails for x minutes from any x of x locations`
+Retry x time before location is marked as failed
+```
+
+Once alert conditions are set, create a message for the alert and specify what services and team members receive the alert notification email and click **Save Test**. You can also use [integrations][5], such as Slack, PagerDuty, webhooks, etc., to receive alert notifications.
+
+### Test results
+
+An **API test** homepage will automatically populate after save. This page includes property information, historical graphs for response time and uptime, sample results, and all events and test results.
+
+To troubleshoot a failed test, scroll to the Test Results section and click on the **Test Results** tab. Click on the failed test, labeled as `Alert`, to view detailed test results. Review the failed assertions and response details such as returned status code, response time, and associated headers and body to resolve the issue.
+
+{{< img src="getting_started/synthetics/api-test-failure.png" alt="API test failure" responsive="true" style="width:90%;">}}
+
+Datadog also has an [APM integration with Synthetics][6] which allows you to go from a test run that failed to the root cause of the issue by looking at the trace generated by that test run.
+
+Alerts triggered with the APM integration populate under **Test Results**. Click the result link given to review test details and traces.
```

Comment:
I would probably remove that one, I feel like it's not super clear what we mean by alerts here.